README.md

```diff
@@ -81,8 +81,7 @@ The resulting addresses should be on all networks the same.
 
 Note: Address will vary if contract code is changed or a different Solidity version is used.
 
-#### Replay protection (EIP-155)
-
+#### Replay protection [(EIP-155)](https://eips.ethereum.org/EIPS/eip-155)
```

Comment:
```suggestion
#### Replay protection ([EIP-155](https://eips.ethereum.org/EIPS/eip-155))

```

Reply:
I think the extra newline makes it easier to read.

Can you please sign the CLA as well?

Reply:
Looking at the creator's contribution history, I believe they might be a bot or an airdrop hunter.

Reply:
> I am not a bot or a hunter, I want to be useful to projects, and for this, no one promises me anything and I do not ask for anything. It's just nice to be thanked for your work, but unfortunately, not everyone understands it. Today I'm looking for bugs and tomorrow I might want to learn how to code, why not. You just need to feel needed, not the other way around

I'm sorry I got it wrong. It's just that most of the time, README changes without any change description are bot farming. Thanks for the change. And also, do learn how to code, it's a great skill.